.editorconfig

```diff
@@ -0,0 +1,31 @@
+# EditorConfig is awesome: http://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+max_line_length = 120
```

Comment:
I think this one is more of a recommendation than a hard requirement and enforcing it will lead to unpleasant results.